src/pages/item/_.vue

```diff
@@ -68,8 +68,7 @@
             class="col-lg-10"
           >
             <MetadataBox
-              :all-metadata="allMetaData"
-              :core-metadata="coreFields"
+              v-model="fieldsAndKeywords"
```

Comment:
What's the reason for using v-model here instead of a named prop? Having the metadataBox component refer to `value[name]` seems a bit odd when `value` is in fact multiple fields, so would make more sense to have `values[name]` or `metadata[name]`

Reply:
Indeed, thank you. That's been changed to `metadata`.